src/export/word/schema.ts

```diff
@@ -116,9 +132,20 @@ export function getNodesAndMarks(styles?: Styles) {
       state.closeBlock(node, styles?.iframe);
     },
     figure(state, node) {
+      const { id, numbered } = node.attrs as Nodes.Figure.Attrs;
+      // TODO: localize this ID
```

Comment:
Can we actually remove this `TODO` @rowanc1 ? What should I check to confirm whether we need to "localize" this or not? (The cross references appeared to be working)

Reply:
We will need an options structure that looks like this:
https://github.com/curvenote/curvenotejs/blob/main/src/export/utils/localizationOptions.ts

The imageSrc isn't needed, but the other ones will be. I think you can actually import these directly from `curvenote`.